src/CGRANode.cpp

```diff
@@ -247,23 +247,23 @@ bool CGRANode::canSupport(DFGNode* t_opt) {
       (t_opt->isStore()      and !canStore()) or
       (t_opt->isReturn()     and !canReturn()) or
       (t_opt->isVectorized() and !supportVectorization()) or
-      (t_opt->isAddSub()        and !canAdd()) or 
-      (t_opt->isMul()        and !canMul()) or 
-      (t_opt->isPhi()        and !canPhi()) or 
-      (t_opt->isSel()        and !canSel()) or 
-      (t_opt->isMAC()        and !canMAC()) or 
-      (t_opt->isLogic()      and !canLogic()) or 
-      (t_opt->isBranch()     and !canBr()) or 
+      (t_opt->isAddSub()     and !canAdd()) or  // We assumes the HW adder can both both add and sub
```

Comment:
`// We assumes the HW adder can both both add and sub` ->
`// We assumes the HW adder can do both add and sub.`